backend/isoko-backend/template.yml

```diff
@@ -281,6 +301,18 @@ Resources:
     Properties:
       AccessControl: PublicRead
       BucketName: image-bucket-isoko
+
+  # Rest Api to use for Cognito authorization
+  CognitoPlatformApi:
+    DependsOn: UserPool
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: CognitoPlatformApi
+      StageName: !Ref Version
```

Comment:
I tried deploying this to my personal AWS account and it looks like Version is not defined in our SAM template anywhere

Here's the error message: `"FAILED" Status: FAILED. Reason: Template format error: Unresolved resource dependencies [Version] in the Resources block of the template`

Reply:
Yeah def an oversight on my part. I'm just going to change it to Prod since we don't have different stages anyway.